src/Pods/Integrations/Service_Provider.php

```diff
@@ -32,8 +36,38 @@ public function register() {
 	 * @since 2.8.0
 	 */
 	protected function hooks() {
+
 		add_filter( 'pods_admin_config_pod_fields_post_type_supported_features', $this->container->callback( 'pods.integration.genesis', 'add_post_type_supports' ) );
 		add_filter( 'pods_admin_config_pod_fields_post_type_supported_features', $this->container->callback( 'pods.integration.yarpp', 'add_post_type_supports' ) );
 		add_filter( 'pods_admin_config_pod_fields_post_type_supported_features', $this->container->callback( 'pods.integration.jetpack', 'add_post_type_supports' ) );
+
+		if ( ! did_action( 'plugins_loaded' ) ) {
+			add_action( 'plugins_loaded', [ $this, 'plugins_loaded' ] );
+		} else {
+			$this->plugins_loaded();
+		}
+	}
+
+	/**
+	 * All plugins are loaded.
+	 *
+	 * @since 2.8.0
+	 */
+	public function plugins_loaded() {
+
+		$active = $this->container->callback( 'pods.integration.polylang', 'is_active' );
+		if ( is_callable( $active ) && $active() ) {
+			$hooks = $this->container->callback( 'pods.integration.polylang', 'hook' );
+			if ( is_callable( $hooks ) ) {
+				$hooks();
+			}
+		}
+		$active = $this->container->callback( 'pods.integration.wpml', 'is_active' );
+		if ( is_callable( $active ) && $active() ) {
+			$hooks = $this->container->callback( 'pods.integration.wpml', 'hook' );
+			if ( is_callable( $hooks ) ) {
+				$hooks();
+			}
+		}
```

Comment:
Let's get this to run off of a filtered array of integrations that we can further add stuff to when we're ready.

With the filter, someone can totally disable one of them too.

Reply:
Use `$this->container->make()` to get the object itself and then call the methods off of that object. `$this->container->callback()` is meant for things not running right away.